### PR TITLE
[sharktank] Enable f8 model perplexity tests

### PR DESCRIPTION
**Pull request by archana-ramalingam**
Enable f8 model perplexity tests in sharktank

### CODE REVIEW

File: sharktank/sharktank/utils/export_artifacts.py [hunk 1]
```diff
@@ -224,6 +228,18 @@ def compile_to_vmfb(
         hal_dump_path: Optional[Path] = None,
         args: Optional[List[str]] = None,
     ):
+        args = [
```
Comment (aviator19941) [inline sharktank/sharktank/utils/export_artifacts.py:231]: Check if args are None, then set them to the list

**Review by aviator19941 — COMMENTED** [review]
Minor comment and seems like some CI's are failing, will approve after they are green.

File: sharktank/sharktank/evaluate/perplexity_iree.py [hunk 2]
```diff
@@ -453,15 +496,16 @@ def main(argv):
...
+    cli.add_tokenizer_options(parser)
+
     args = cli.parse(parser, args=argv)
 
     torch_device = torch.device(args.device) if args.device else None
     weight_path = cli.get_input_dataset(args)
     tokenizer = cli.get_tokenizer(args)
 
-    use_attention_mask = True
+    use_attention_mask = False
```
Comment (AmosLewis) [inline sharktank/sharktank/evaluate/perplexity_iree.py:508]: Shouldn't this set with flag --use-attention-mask, this flag set in export cmd in https://github.com/nod-ai/shark-ai/pull/907

Comment (archana-ramalingam) [reply sharktank/sharktank/evaluate/perplexity_iree.py:508]: Ideally we do not want to pass the attention_mask explicitly in sharktank. But will add a flag for now.

Comment (archana-ramalingam) [reply sharktank/sharktank/evaluate/perplexity_iree.py]: Done

File: sharktank/sharktank/utils/export_artifacts.py [hunk 3]
```diff
@@ -318,6 +336,11 @@ def create_file(self, *, suffix, prefix):
 
     def get_artifacts(self):
 
+        assert self.attention_kernel in [
+            "decomposed",
+            "torch",
+        ], "Only torch or decomposed attention_kernel types are supported"
```
Comment (AmosLewis) [inline sharktank/sharktank/utils/export_artifacts.py:342]: ` choices=["decomposed", "torch", "sharktank"],`
https://github.com/nod-ai/shark-ai/pull/907/files#diff-3472e6cbe791396cfe79f956d3c3df7aa365418090ed67f47cf61bebf16ab5ebR78 sharktank/sharktank/utils/cli.py 

A new attention kernel added in Dan's atten_f8 pr.

Comment (archana-ramalingam) [reply sharktank/sharktank/utils/export_artifacts.py:342]: Probably will make sense to add the kernel in Dan's PR.

**Review by aviator19941 — APPROVED** [review]
